pkg/resources/samples/trace-socket-collector.yaml

```
@@ -0,0 +1,13 @@
+apiVersion: gadget.kinvolk.io/v1alpha1
+kind: Trace
+metadata:
+  name: socket-collector
+  namespace: gadget
+spec:
+  node: my-node
+  gadget: socket-collector
+  filter:
+    namespace: default
+    podname: myapp1-pod-wsgx9
```

```suggestion
    podname: mypod
```

? 

Just to avoid putting a pod name what nobody will ever use.